2020q2/bhyve-intel-openbsd.md

```diff
@@ -0,0 +1,56 @@
+## PCI passthrough with bhyve on Intel and for OpenBSD guests
+
+Link:	 [bhyve Intel bug report](https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=229852)  
+Link:	 [bhyve OpenBSD bug report](https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=245392)  
+Link:	 [PCI passthrough with bhyve (FreeBSD wiki article)](https://wiki.freebsd.org/bhyve/pci_passthru)  
+
+Contact: Anatoli <me@anatoli.ws>  
+Contact: Callum <callum@aitchison.org>  
+Contact: Peter Grehan <grehan@freebsd.org>  
+
+bhyve(8) is a hypervisor that supports running a variety of guest
+operating systems in virtual machines. bhyve(8) includes support for PCI
+devices passthru, a technique to pass host PCI devices to a virtual
+machine for its exclusive control and use.
+
+For some years, PCI passthrough (ppt) in bhyve was not working on some
+Intel systems and for OpenBSD guests due to two bugs. The first one was
+crashing FreeBSD host when bhyve was started with ppt on Intel
+processors with two VT-d translation units (IOMMU), included in most
+Skylake and newer Intel processors.
+
+The second bug was preventing correct interrupts handling for OpenBSD
+guests. As a result, OpenBSD guests running on bhyve were not able to
+use any PCI devices passed through to them from the host.
+
+During the last 2 months the second bug was identified and fixed and
+they both were backported to 12.1-RELEASE (p7). So now it's possible to
+fully take advantage of PCI passthrough with bhyve in a production-ready
```

Comment:
Shortening PCI passthrough here is fine, but perhaps you should add a parenthesis mentioning the shortened form you use later?

Reply:
Do you mean something like this: "So now it's possible to fully take advantage of PCI passthrough (ppt) with bhyve in a production-ready..."?